packages/design-system/src/forms/FormikPinField.tsx

```
@@ -20,68 +32,83 @@ export interface FormikPinFieldProps {
+              .fill('')
+              .map((_, index) => (
+                <InputOTPSlot
+                  index={index}
+                  data-cy={`${data?.cy}-${index + 1}`}
+                  data-test={`${data?.test}-${index + 1}`}
+                  className={twMerge(
+                    'h-9 text-base',
+                    !!meta.error && meta.touched && 'border-y border-red-600',
+                    className?.inputItem
+                  )}
+                />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)